src/main/java/com/github/jsonapi/retrofit/JSONAPIConverterFactory.java

```diff
@@ -18,6 +18,11 @@
 public class JSONAPIConverterFactory extends Converter.Factory {
 	private ResourceConverter parser;
 
+	public JSONAPIConverterFactory(ResourceConverter parser)
+	{
```

Comment:
Sure, will do. Accidentally used my formatting style instead of the style from your code.